engine/classes/Elgg/ViewsService.php

```diff
@@ -745,12 +646,16 @@ public function getInspectorData(): array {
 				$overrides = $data;
 			}
 		}
-
+		
+		$simplecache = _elgg_services()->simpleCache;
+		$reflector = new \ReflectionClass($simplecache);
+		$simplecache_views = $reflector->getProperty('simplecache_views')->getValue($simplecache);
```

Comment:
aahh, why not using a new public function? declare the function `@internal` and be done with it

Reply:
done